salt/metalk8s/kubeadm/init/kubeconfig/lib.sls

```diff
@@ -0,0 +1,28 @@
+{%- from "metalk8s/map.jinja" import defaults with context %}
+
+{% macro kubeconfig(name, cert_info) %}
+
+{%- set ca_server = salt['mine.get']('*', 'kubernetes_ca_server').keys() %}
+{%- set apiserver = 'https://' ~ salt['grains.get']('master') ~ ':6443' %}
+
+{%- if ca_server %}
+
+Create kubeconf file for {{ name }}:
+  metalk8s_kubeconfig.managed:
+    - name: /etc/kubernetes/{{ name }}.conf
+    - ca_server: {{ ca_server[0] }}
+    - signing_policy: kube_user_account_policy
```

Comment:
Seems like my PR will get merged first you will need to change this to use `kube_api.cert.client_signing_policy` value from map.jinja